src/components/SQFormDialogStepper/SQFormDialogStepper.tsx

```diff
@@ -12,66 +11,119 @@ import {
+  props: TransitionProps & {children?: React.ReactElement<unknown, string>},
+  ref: React.Ref<unknown>
+) {
   return <Slide direction="down" ref={ref} {...props} />;
 });
 
 const useStyles = makeStyles({
   root: {
     padding: 20,
     width: '100%',
-    maxWidth: ({steps}) => `${300 * steps.length}px`,
+    maxWidth: ({
+      steps,
+    }: {
+      steps: Array<React.ReactChild | React.ReactFragment | React.ReactPortal>;
+    }) => `${300 * steps.length}px`,
     '& svg': {
       fontSize: 30,
       '& text': {
         fontSize: 15,
-        fontWeight: 600
-      }
+        fontWeight: 600,
+      },
     },
     '& span': {
-      whiteSpace: 'nowrap'
+      whiteSpace: 'nowrap',
     },
     '& .MuiStepLabel-label.MuiStepLabel-alternativeLabel': {
-      marginTop: '5px'
-    }
-  }
+      marginTop: '5px',
+    },
+  },
 });
 
 const useActionsStyles = makeStyles({
   root: {
     display: 'flex',
     justifyContent: 'space-between',
     flex: '1 1 100%',
-    padding: '16px 24px'
-  }
+    padding: '16px 24px',
+  },
 });
 
 const useStepperStyles = makeStyles({
   root: {
     padding: '1px',
-    justifyContent: 'center'
-  }
+    justifyContent: 'center',
+  },
 });
 
+interface SQFormDialogStepperProps {
+  /** The secondary button text (Button located on left side of Dialog) */
+  cancelButtonText?: string;
+  /** The content to be rendered in the dialog body.  Will be an array of React elements. */
+  children: Array<JSX.Element>;
+  /** If true, clicking the backdrop will not fire the onClose callback. */
+  disableBackdropClick?: boolean;
+  /** Sets the dialog to the maxWidth. */
+  fullWidth?: boolean;
+  /** Disables the next/submit button */
+  isNextDisabled?: boolean;
+  /** The current open/closed state of the Dialog */
+  isOpen: boolean;
+  /** Allows the initial values to be updated after initial render */
+  enableReinitialize?: boolean;
+  /** Determine the max-width of the dialog. The dialog width grows with the size of the screen. Set to false to disable maxWidth. */
+  maxWidth?: 'xs' | 'sm' | 'md' | 'lg' | 'xl' | false;
+  /** Callback function invoked when the user clicks on the secondary button or outside the Dialog */
+  onClose: () => void;
+  /** Callback function invoked when user clicks primary submit button */
+  onSubmit: (values: Record<string, unknown>, helpers: unknown) => void;
+  /** Title text at the top of the Dialog */
+  title: string;
+  /** Form Entity Object */
+  initialValues: Record<string, string>;
+  /** Callback function that is called when a step is completed to pass back the current state values to the consumer */
+  setValues?: (values: Record<string, unknown>) => void;
+  /** Any prop from https://material-ui.com/api/grid */
+  muiGridProps?: GridProps;
+  /** Any prop from https://material-ui.com/api/dialog/#props */
+  dialogProps?: DialogProps;
+  /** Optional styling on the dialog */
+  contentStyle?: Record<string, unknown>;
```

Comment:
We can likely get the type for this from `DialogContentProps['style']` to ensure we are only giving keys that are actual styles and have proper values